docs/migration-guides/MIGRATION_GUIDE_v5_to_v6.md

```diff
@@ -185,3 +185,45 @@ The custom JUnit extensions have been removed. Please make use of the
 official [JUnit5 extension](https://github.com/weld/weld-testing/tree/master/junit5), which this
 module is now providing. An example how
 to use it can be found in [sda-commons-server-weld-example](../../sda-commons-server-weld-example).
+
+## Automation
+
+The following bash script can help you to quickly migrate your project to sda-dropwizard-commons 6.
+Copy the content to a file in the root of your project and execute it.
+
+```bash
+touch migrate.sh
+chmod u+x migrate.sh
+````
+
+```bash
+#!/bin/bash
+
+echo "Migrating Java files"
+for file in `find . -type f -name "*.java"`
+do
+  echo "Processing $file"
+  sed -i '' -e 's/javax.validation/jakarta.validation/g' $file
+  sed -i '' -e 's/javax.ws.rs/jakarta.ws.rs/g' $file
```

Comment:
Ran this in the Partner ODS and this might come in handy, too:
```suggestion
  sed -i '' -e 's/javax.ws.rs/jakarta.ws.rs/g' $file
  sed -i '' -e 's/javax.annotation/jakarta.annotation/g' $file
```